cypress/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "include": ["**/*.ts"],
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    "downlevelIteration": true,
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    "module": "esnext" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
+    "lib": [
+      "DOM",
+      "ES2020"
+    ] /* Specify library files to be included in the compilation. */,
+    "allowJs": true /* Allow javascript files to be compiled. */,
+    "noEmit": true,
+    "sourceMap": true /* Generates corresponding '.map' file. */,
+    "strict": true /* Enable all strict type-checking options. */,
+    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
+    "types": ["cypress", "@testing-library/cypress"],
+    "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
+    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
+    "resolveJsonModule": true,
+    "skipLibCheck": true /* Skip type checking of declaration files. */,
+    "isolatedModules": false,
+    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
+  }
+}
```

Comment:
Can we define these options in the root `tsconfig.json`, and then just `extend` it here?
For example, in my project's `cypress/`, we have:
```json
{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    "noEmit": false,
    "esModuleInterop": true
  },
  "include": ["../global.d.ts", "../node_modules/cypress", "../node_modules/cypress-xpath", "*/*.ts"]
}

```

Reply:
That's what I wanted to do. But currently there seem to be separate tsconfigs for every package. Just trying to be consistent.